views/score.erb

```diff
@@ -0,0 +1,19 @@
+<a href="/">Home</a>
```

Comment:
Thanks for the reference. I've started implementing nav bars have been handy in projects since this one.